js/remove_anchor_from_url.js

```diff
@@ -0,0 +1,8 @@
+// https://www.codewars.com/kata/remove-anchor-from-url/train/javascript
+
+function removeUrlAnchor(url) {
+    const urlSplitted = url.split('#');
```

Comment:
Nice :+1: 

Are there alternative ways to achieve this? This one is correct, but it is worth thinking if there are other ways

Reply:
I will prepare other solutions and create new PR.